unicodetools/src/test/java/org/unicode/text/UCD/TestTestCodeInvariants.java

```diff
@@ -0,0 +1,19 @@
+package org.unicode.text.UCD;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class TestTestCodeInvariants {
```

Comment:
These two TestTest... classes seem awkward.
I see in the PR description "Does not refactor code into the test area, yet" -- is that the next step for this PR? Moving the regular files into the test path and changing them directly, to JUnit tests, replacing these files?

Reply:
@markusicu it's because there are other functions in the original class.  Is it called from elsewhere, is it run directly with a parameter? If I remove the original class from `src/main/java` then it can't be called as easily.

If this looks right I can merge this and then reduce the test classes in a future PR.